src/Tree.hpp

```diff
@@ -135,6 +140,30 @@ namespace RRT
 
 
         /**
+         * A coefficient that determines how much we want to expand our stepsize when adaptive stepsize control.
+         */
+        int ascLimit() const {
+            return _ascLimit;
+        }
+        void setASCLimit(int lim) {
+            _ascLimit = lim;
+        }
+
+
+        /**
+         * Whether or not the tree is to run with adaptive stepsize control.
+         */
+        bool isDynamic() const {
```

Comment:
It might be clearer if this getter is named just like the setter.  Maybe `usesASC()` or `isAscEnabled()` or something similar.